src/Components/Tabs.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import { TabItem } from './TabItem';
+import { Tab } from '../types/Tab';
+
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
```

Comment:
I thinks it's better to move data to separate file

Reply:
@ivankhrsh in this case, I think it makes no sense to move it to separate file